doc/api/crypto.md

```diff
@@ -42,11 +42,13 @@ calling `require('crypto')` will result in an error being thrown.
 When using CommonJS, the error thrown can be caught using try/catch:
 
 ```cjs
-let crypto;
-try {
-  crypto = require('crypto');
-} catch (err) {
-  console.log('crypto support is disabled!');
+{
+  let crypto;
+  try {
+    crypto = require('crypto');
+  } catch (err) {
+    console.log('crypto support is disabled!');
+  }
```

Comment:
Sorry, could you elaborate on this change?

Reply:
Without this change, the linter fails saying that modifying the global is not allowed. If you prefer we can also disable the linter for this snippet.

Reply:
> If you prefer we can also disable the linter for this snippet.

I think that's the way to go.